code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -113,6 +113,12 @@
 	///whether AI is anchored or not, used for checks
 	var/is_anchored = TRUE
 
+	///Command report cooldown
+	COOLDOWN_DECLARE(command_report_cd) // monkestation edit
+
+	// monkestation edit: uwu-speak upgrade
+	var/datum/fluffy_tongue = FALSE
```

Comment:
this can be modularized, also not the right way to do comments. Read the modularization README please